cmd/clusterctl/cmd/root.go

```diff
@@ -48,12 +51,29 @@ func Execute() {
 func init() {
 	flag.CommandLine = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
 
-	verbosity := flag.CommandLine.Int("v", 0, "Set the log level verbosity.")
-	logf.SetLogger(logf.NewLogger(logf.WithThreshold(verbosity)))
+	verbosity = flag.CommandLine.Int("v", 0, "Set the log level verbosity.")
```

Comment:
Did you mean to remove the additional text about the `CLUSTERCTL_LOG_LEVEL` from the previous commit? I kinda liked that there. 🙂

Reply:
🙂